src/App/Header/MobileNav/Menu.tsx

```diff
@@ -36,6 +36,8 @@ export default function Menu({
               key={`header-link-${link.title}`}
               className={styles}
               href={link.href}
+              target="_blank"
```

Comment:
we could use `components/ExtLink` for this

Reply:
Good call! :) Will swap that in.

Reply:
Fixed in 573ab7a78.